packages/dao/test/api3template.js

```diff
@@ -1,120 +1,57 @@
-
-    daoFactory =  await DaoFactoryFactory.deploy(
-        kernel.address,
-        acl.address,
-        evmScriptRegistryFactory.address
-    );
-
-    ens =  await ENSFactory.deploy();
-
-    const publicResolver = await PublicResolverFactory.deploy(ens.address);
-
-    miniMeTokenFactory =  await MiniMeTokenFactoryFactory.deploy();
-
-    fifsResolvingRegistrar =  await FIFSResolvingRegistrarFactory.deploy(
-        ens.address,
-        publicResolver.address,
-        '0x0000000000000000000000000000000000000000000000000000000000000000'
-    );
-});
-
-describe("api3template tests", function () {
-    context("creation of the api3", function () {
-        it("construction revert because of contracts addresses", async () => {
-
-            try {
-                await api3TemplateFactory.deploy(
-                    "0x0000000000000000000000000000000000000000",
-                    ens.address,
-                    miniMeTokenFactory.address,
-                    fifsResolvingRegistrar.address
-                );
-                throw new Error("Test without daoFactory works");
-            } catch(error) {
-                expect(error.toString()).to.equal("Error: VM Exception while processing transaction: revert TEMPLATE_DAO_FAC_NOT_CONTRACT");
-            }
-
-            try {
-                await api3TemplateFactory.deploy(
-                    daoFactory.address,
-                    "0x0000000000000000000000000000000000000000",
-                    miniMeTokenFactory.address,
-                    fifsResolvingRegistrar.address
-                );
-                throw new Error("Test without ENS works");
-            } catch(error) {
-                expect(error.toString()).to.equal("Error: VM Exception while processing transaction: revert TEMPLATE_ENS_NOT_CONTRACT");
-            }
-
-        });
-
-        it("construction completes correctly", async () => {
-            await api3TemplateFactory.deploy(
-                daoFactory.address,
-                ens.address,
-                miniMeTokenFactory.address,
-                fifsResolvingRegistrar.address
-            );
-        });
-
-    });
-});
+/*global artifacts, web3, contract, before, assert*/
+/*eslint no-undef: "error"*/
+
+// const { APP_IDS } = require('@aragon/templates-shared/helpers/apps')
+const { assertRole } = require('@aragon/templates-shared/helpers/assertRole')(web3)
+const { getEventArgument } = require('@aragon/test-helpers/events')
+const { /*getENS,*/ getTemplateAddress } = require('@aragon/templates-shared/lib/ens')(web3, artifacts)
+// const { getInstalledAppsById } = require('@aragon/templates-shared/helpers/events')(artifacts)
+
+const ACL = artifacts.require('ACL')
+const Kernel = artifacts.require('Kernel')
+const Voting = artifacts.require('Voting')
+const Api3Template = artifacts.require('Api3Template')
+
+contract('Api3Template', ([_, deployer, tokenAddress, authorized]) => { // eslint-disable-line no-unused-vars
```

Comment:
What about tests which revert on construction?

Reply:
I don't think that it's crucial, we can add in a different PR later on though